files/image_config/config-setup/config-setup

```diff
@@ -355,6 +362,10 @@ do_config_backup()
     echo "Taking backup of curent configuration"
     rm -rf /host/old_config
     cp -ar /etc/sonic /host/old_config
+
+    # Save TACACS
+    sonic-cfggen -d --preset tac > /host/old_config/${TACACS_JSON_BACKUP}
```

Comment:
i am not sure if we need this as the config_db has the tacacs cofiguration.

Reply:
dropped